anaconda.py

```diff
@@ -457,10 +457,11 @@ def _earlyExceptionHandler(ty, value, traceback):
         flags.ksprompt = False
 
     # Set minimal ram size to the storage checker.
+    from pyanaconda.core import hw
     if anaconda.display_mode == constants.DisplayModes.GUI:
-        min_ram = isys.MIN_GUI_RAM
+        min_ram = hw.MIN_GUI_RAM
     else:
-        min_ram = isys.MIN_RAM
+        min_ram = hw.MIN_RAM
```

Comment:
Could you move this code + the storage checker below to a new function in `startup_utils`, please?

Reply:
Added.